### DISCUSSION TITLE
Application icons in the overview

### DISCUSSION THREAD

**Discussion by NikitaPZ** [end-4/dots-hyprland#797] [Q&A]
Anyone know a way for overview to pick up all the icons for the applications I install on my system?

**Comment by end-4** [comment]
it's impossible for the overview to perfectly show icon of every app, since the icon name can be different to the window class that hyprland reports. 
you can use `hyprctl clients` to see what the class name is and try to add a window class substitution for it in the user options that substitutes the window class to the correct icon name in your icon pack

**Comment by lifeashansen** [comment]
Thanks. This helped
